yt/docs/ru/user-guide/query-tracker.md

```diff
@@ -0,0 +1,158 @@
+# Query tracker
+
+Query tracker позволяет:
+
++ Отправлять произвольные запросы на исполнение.
++ Отслеживать исполнение.
++ Сохранять результаты.
++ Смотреть историю запросов.
+
+Запросы характеризуются:
+
++ Движком.
++ Текстом.
+
+Движок запроса контролирует исполнение запроса.
+
+Текст запроса специфичен для движка.
+
+## Движки
+
+На текущий момент поддерживаются следующие движки исполнения:
+
++ YT QL
+  + Простой движок, поддерживающий простые запросы вида `* from [//home/user/table]`
++ [YQL](../yql/index.md)
+  + Запрос исполняется в YQL агентах. Они разбивают запрос в YT операции (map, reduce, ...), запускают их, собрают и возвращают результат.
++ [CHYT](./data-processing/chyt/about-chyt.md)
+  + Запрос исполняется в клике.
++ [SPYT](./data-processing/spyt/overview.md)
+  + Запрос исполняется в spark кластере.
+
+## API
+
+Все операции принимают параметр `query_tracker_stage`, который позволяет выбрать HELP NEEDED HERE.
+
+### start_query
+
+Отправляет запрос на исполнение.
+
+Обязательные параметры:
+
++ `engine` - движок для исполнения запроса. Поддерживаются `ql`, `yql`, `chyt`, `spyt`
++ `query` - текст запроса.
+
+Опциональные параметры:
+
++ `files` - список файлов для запроса в формате YSON.
++ `stage` - query tracker stage.
++ `settings` - дополнительные параметры для запроса в формате YSON.
```

Comment:
It may be useful to describe the most specific ones. For example, that a chyt clique should be written here